### PR TITLE
Logging spdlog

### PR DESCRIPTION
**Pull request by aymanhab**
Fixes issue #0

### Brief summary of changes
Change Logging to Messages window of the application to use the new logging system.
### CHANGELOG.md (choose one)

- no need to update because...
- updated...

### CODE REVIEW

**Comment by aymanhab** [comment]
@chrisdembia This branch should work with the feature logging branch.

**Comment by chrisdembia** [comment]
@aymanhab after I fixed an issue in opensim-core (see https://github.com/opensim-org/opensim-core/pull/2754), logging in the GUI worked:

image

Note: I temporarily added "DEBUGLOGGING" to the messages to ensure the message in the GUI was coming from the JavaLogSinkToMessages:

```
LoggerTopComponent.findInstance().log("DEBUGLOGGING" + msgFinal);
```

File: Gui/opensim/logger/src/org/opensim/logger/JavaLogSinkToMessages.java [hunk 1]
```diff
@@ -28,7 +28,7 @@
 import org.opensim.modeling.LogSink;
 import org.opensim.modeling.Logger;
 
-public class JavaLogSinkToMessages extends JavaLogSink {
+public class JavaLogSinkToMessages extends LogSink {
```
Comment (chrisdembia) [inline Gui/opensim/logger/src/org/opensim/logger/JavaLogSinkToMessages.java:31]: Oh, yes, this is more correct.

**Comment by aymanhab** [comment]
Thanks @chrisdembia Will merge after feature_logging branch is merged into opensim-core master.

**Comment by aymanhab** [comment]
Now that opensim-core side is using spdlog, merging this PR that brings GUI in sync. Thanks @chrisdembia

**Pull request merged**